src/test/java/edu/cornell/kfs/rass/RassTestConstants.java

```diff
@@ -14,6 +14,7 @@ private RassTestConstants() {
     public static final String DEFAULT_PROPOSAL_AWARD_TYPE = "S";
     public static final Integer DEFAULT_DD_FIELD_MAX_LENGTH = Integer.valueOf(50);
     public static final String ERROR_OBJECT_KEY = "ERR";
+    public static final String ERROR_OBJECT_KEY_FOR_EXPECTED_RESULTS = "Agency " + ERROR_OBJECT_KEY;
```

Comment:
Should this be named differently to reflect this is for Agency expected results? 
Meaning would we possible need this type of constant for Award, Proposal, etc. in the future where the name of the constant is too generic for the specific error that is currently being trapped?

Reply:
Yes, I will rename to signify it is Agency specific